<?xml version="1.0" encoding="utf-8"?>
<!-- This URDF was automatically created by SolidWorks to URDF Exporter! Originally created by Stephen Brawner (brawner@gmail.com) 
     Commit Version: 1.5.1-0-g916b5db  Build Version: 1.5.7152.31018
     For more information, please see http://wiki.ros.org/sw_urdf_exporter -->
<robot
  name="inchworm_description">
  <link
    name="base_link">
    <inertial>
      <origin
        xyz="-0.0016917 -0.0012554 0.063867"
        rpy="0 0 0" />
      <mass
        value="0.10777" />
      <inertia
        ixx="3.8554E-05"
        ixy="9.4075E-07"
        ixz="1.709E-06"
        iyy="4.445E-05"
        iyz="1.2246E-06"
        izz="2.5729E-05" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://inchworm_description/meshes/base_link.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="1 1 1 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://inchworm_description/meshes/base_link.STL" />
      </geometry>
    </collision>
  </link>
  <link
    name="link_1">
    <inertial>
      <origin
        xyz="0.0033121 -0.0010932 -0.10202"
        rpy="0 0 0" />
      <mass
        value="0.12376" />
      <inertia
        ixx="5.6824E-05"
        ixy="-9.4075E-07"
        ixz="5.513E-07"
        iyy="7.822E-05"
        iyz="-1.2246E-06"
        izz="3.8675E-05" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://inchworm_description/meshes/link_1.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="1 1 1 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://inchworm_description/meshes/link_1.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="joint_1"
    type="continuous">
    <origin
      xyz="0 -0.0002 0.10478"
      rpy="-3.1399 0 3.1416" />
    <parent
      link="base_link" />
    <child
      link="link_1" />
    <axis
      xyz="1 0 0" />
    <limit
      lower="0"
      upper="0"
      effort="0"
      velocity="0" />
  </joint>
  <link
    name="link_2">
    <inertial>
      <origin
        xyz="0.026489 0.013907 -0.11702"
        rpy="0 0 0" />
      <mass
        value="0.12376" />
      <inertia
        ixx="5.6824E-05"
        ixy="-9.4075E-07"
        ixz="5.513E-07"
        iyy="7.822E-05"
        iyz="-1.2246E-06"
        izz="3.8675E-05" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://inchworm_description/meshes/link_2.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="1 1 1 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://inchworm_description/meshes/link_2.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="joint_2"
    type="continuous">
    <origin
      xyz="-0.02165 -0.015204 -0.14832"
      rpy="0.00024788 0 0" />
    <parent
      link="link_1" />
    <child
      link="link_2" />
    <axis
      xyz="-1 0 0" />
    <limit
      lower="0"
      upper="0"
      effort="0"
      velocity="0" />
  </joint>
  <link
    name="link_3">
    <inertial>
      <origin
        xyz="0.025131 0.015 -0.071936"
        rpy="0 0 0" />
      <mass
        value="0.075697" />
      <inertia
        ixx="2.7758E-05"
        ixy="9.4654E-18"
        ixz="-1.1577E-06"
        iyy="4.3258E-05"
        iyz="7.0031E-19"
        izz="3.2746E-05" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://inchworm_description/meshes/link_3.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="1 1 1 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://inchworm_description/meshes/link_3.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="joint_3"
    type="continuous">
    <origin
      xyz="0.0017 -0.00017056 -0.16329"
      rpy="-0.0019608 0 0" />
    <parent
      link="link_2" />
    <child
      link="link_3" />
    <axis
      xyz="-1 0 0" />
    <limit
      lower="0"
      upper="0"
      effort="0"
      velocity="0" />
  </joint>
</robot>